datapunt_api/rest.py

```diff
@@ -1,93 +1,11 @@
-from rest_framework import viewsets
-from rest_framework import renderers
-from rest_framework_extensions.mixins import DetailSerializerMixin
-from django_filters.rest_framework import DjangoFilterBackend
+""" Will be deprecated soon """
```

Comment:
Expand docstring to explain which classes you should import in stead.

Also: expose `_DisabledHTMLFilterBackend` as before. We are not sure there are no users (although it was marked as private).

Reply:
`_is_detailed_request` is also gone. Should not have been used but ...